src/material-experimental/mdc-button/README.md

```diff
@@ -1,3 +1,78 @@
-### TODO
-- Consider supporting the selector "button[mat-outlined-button]" for the stroked button
-- Consider if we want to add a "appearance/type" input to button, similar to form field (outline/stroked, flat, raised)
+This is a prototype of an alternate version of `MatButton` built on top of
+[MDC Web](https://github.com/material-components/material-components-web). This component is
+experimental and should not be used in production.
+
+## How to use
+Assuming your application is already up and running using Angular Material, you can add this 
+component by following these steps:
+
+1. Install `@angular/material-experimental` and MDC Web:
+
+   ```bash
+   npm i material-components-web @angular/material-experimental
+   ```
+
+2. In your `angular.json`, make sure `node_modules/` is listed as a Sass include path. This is
+   needed for the Sass compiler to be able to find the MDC Web Sass files.
+
+   ```json
+   ...
+   "styles": [
+     "src/styles.scss"
+   ],
+   "stylePreprocessorOptions": {
+     "includePaths": [
+       "node_modules/"
+     ]
+   },
+   ...
+   ```
+
+3. Import the experimental `MatButtonModule` and add it to the module that declares your component:
+
+   ```ts
+   import {MatButtonModule} from '@angular/material-experimental/mdc-button';
+
+   @NgModule({
+     declarations: [MyComponent],
+     imports: [MatButtonModule],
+   })
+   export class MyModule {}
+   ```
+
+4. Use the buttons in your component's template:
+   
+   ```html   
+   <button mat-button> Basic </button>
+   <button mat-raised-button> Raised </button>
+   <button mat-stroked-button> Stroked </button>
+   <button mat-flat-button> Flat </button>   
+   <button mat-icon-button>
+     <mat-icon>save</mat-icon>
+   </button>
+   <button mat-fab>
+     <mat-icon>add</mat-icon>
+   </button>
+   ``` 
+   
+5. Add the theme and typography mixins to your Sass. Note that there are three separate mixins for 
+the button variants: standard buttons, icon buttons, and the FAB. Include only the mixins of the 
```

Comment:
Nit: should this be `and floating action buttons` (or FABs). The other variants are plural while this is about _the_ fab 😄 

<sup>Non-native english so happy to learn something new 😄</sup>

Reply:
+1 thanks, that makes a lot more sense